force-app/main/default/classes/EmailUtility.cls

```diff
@@ -0,0 +1,36 @@
+public class EmailUtility {
+    public static void toSendEmailWithTemplate(String Name, String objectType){
+        List<Messaging.SingleEmailMessage> mailList = new List<Messaging.SingleEmailMessage>();
+        
+        EmailTemplate emailTemp = [SELECT Id,
+                                  Subject,
+                                  Body
+                                  FROM EmailTemplate
+                                  WHERE DeveloperName = 'Decision_Maker'];
+        
+        try{
+            sObject targetObject;
+        if(objectType == 'Contact'){
+            targetObject = [SELECT Id, Email FROM Contact WHERE LastName =: Name];
+        }else if(objectType == 'User'){
```

Comment:
According to the mentioned above (using SObjectType class), change should be "objectType == User.SObjectType"